### DISCUSSION TITLE
How to send renderView content to a new browser tab

### DISCUSSION THREAD

**Discussion by cfcoder** [wheels-dev/wheels#1395] [Q&A]
For a report section of my site, I have a link that references a route to an action that calls renderView() where a template is specified. In that template file is the code to generate a  PDF (or Excel file). Previous code used an onclick JS call to a JS function that used a window.open(url) call.  This opens the template in a new browser tab.  Now I'm wanting to use a route in the link (using urlFor() to render the path) and the action call the PDF generating template file, but in a new tab, as before. I'm not seeing a Wheels way to do that. I would think the renderView would have a target attribute (target="_blank") to allow this, but it doesn't work nor is that documented anywhere.

I also tried using redirectTo which says it allows passing in params but adding this attribute (params='target="_blank"') to redirectTo() did nothing. Just opens the path template replacing the existing path.

My objective is to list some links to trigger generating reports that open in a NEW tab, not the existing tab.

How do others get around this?

**Comment by chapmandu** [comment]
I'd use `#linkTo(route = "myRoute", target="_blank", params="foo=bar&thisargumentisnotrequired=1")#`
